src/core/resources/transactions/transaction.ts

```diff
@@ -48,16 +61,25 @@ export const fetchTransaction = async ({
     });
     const tx = response.data.payload.transaction;
     if (response.data.meta.status === 'pending') {
+      const localPendingTx = searchInLocalPendingTransactions(address, hash);
```

Comment:
why do we want to do this?

i'm thinking that we need an updated state of the tx if BE has it as pending

Reply:
backend doesn't return all the tx data when it's pending, and the `getCustomChainTransaction` overrides the transaction type to `send` as it can't know that metadata
I don't see what new data we could grab from a pending tx like when it completes/reverts we update

Reply:
ok went through the code and this is only fetching tx from be i was thinking this could be used for custom networks txs